app/views/canvas/edit/_my_module.html.erb

```diff
@@ -23,40 +23,40 @@
         <li class="dropdown-header"><%= t('projects.index.options_header') %></li>
         <% if can_edit_module(my_module) %>
           <li>
-              <a class="edit-module" href="" data-module-id="<%= my_module.id %>"><%=t "projects.canvas.edit.edit_module" %></a>
+              <a class="edit-module" href="" data-module-id="<%= my_module.id %>"><%=t "experiments.canvas.edit.edit_module" %></a>
           </li>
         <% end %>
-        <% if can_edit_module_groups(my_module.project) %>
+        <% if can_edit_module_groups(my_module.experiment.project) %>
```

Comment:
I'd also refactor those permissions in this file to take experiment as an argument. Sorry for work :anguished:.